src/presentation/__tests__/presentation.test.ts

```
@@ -0,0 +1,25 @@
+import { buildApp } from '../index';
+import { FastifyInstance } from 'fastify';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)